apps/modernization-ui/src/pages/addPatient/AddPatient.tsx

```diff
@@ -100,8 +100,25 @@ const AddPatient = () => {
         setEntryState({ step: 'entry' });
     };
 
+    function isMissingFields(entry: NewPatientEntry) {
+        for (const key in entry) {
+            if (Object.hasOwnProperty.call(entry, key)) {
+                const value = entry[key as keyof NewPatientEntry];
+                if (
+                    value === null ||
+                    value === undefined ||
+                    (typeof value === 'string' && value === '') ||
+                    (Array.isArray(value) && value.length === 0)
+                ) {
+                    return false; // At least one property has no value
+                }
+            }
+        }
+        return true;
+    }
+
     const evaluateMissingFields = (entry: NewPatientEntry) => {
-        setEntryState({ step: 'verify-missing-fields', entry });
+        setEntryState({ step: isMissingFields(entry) ? 'create' : 'verify-missing-fields', entry });
```

Comment:
This is by passing the Address Verification. 

If the `entry` is missing fields it should call `setEntryState({step: 'verify-missing-fields', entry})` to prompt the user if they would like to continue.

If the `entry` is not missing fields it should skip the prompt and just call `setEntryState({step: 'verify-address', entry})`.